adapter/integration-test/src/adapter-datasource-trigger.test.js

```diff
@@ -151,22 +145,22 @@ describe('Adapter Sources Trigger', () => {
     })
   })
 
-  test('POST datasource/{id}/trigger FAIL', async () => {
+  test('Should publish results for failing datasources [POST /datasources/{id}/trigger]', async () => {
     const brokenDatasourceConfig = Object.assign({}, staticDatasourceConfig)
     brokenDatasourceConfig.protocol.parameters.location = 'LOL'
-    const datasourceResponse = await request(URL)
+    const datasourceResponse = await request(ADAPTER_URL)
       .post('/datasources')
       .send(brokenDatasourceConfig)
 
     const datasourceId = datasourceResponse.body.id
 
-    const triggerResponse = await request(URL)
+    const triggerResponse = await request(ADAPTER_URL)
       .post(`/datasources/${datasourceId}/trigger`)
       .send()
 
     expect(triggerResponse.status).toBeGreaterThan(300) // request should fail (no 2xx status)
```

Comment:
What status code is here really expected?
It can't be the case that a random error status code is returned.

Reply:
The status code returned is 404 which is not the behavior we want. IMO it should be 500. The root cause is addressed in issue #156. 
I agree that, when #156 is done, we should specify the response code. Until then, however, I would like to leave it open, since expecting 404 is not correct either.